src/libstore/unix/build/derivation-builder.cc

```diff
@@ -3056,5 +2097,85 @@ StorePath DerivationBuilderImpl::makeFallbackPath(const StorePath & path)
         Hash(HashAlgorithm::SHA256), path.name());
 }
 
+}
+
+// FIXME: do this properly
+#include "linux-derivation-builder.cc"
+#include "darwin-derivation-builder.cc"
```

Comment:
I'll let this slide for now :)

Reply:
Out of curiosity, what would "do this properly" look like? A separate file that has these includes and the function definition, or...?

Reply:
Yeah, we'd have to move `DerivationBuilderImpl` into a header. But putting an impl class in a header kind of defeats the purpose. And there are probably some more refactorings that we want to do (like splitting `DerivationBuilder` and `UnixDerivationBuilder`) but we can do that in a future PR.